### ISSUE TITLE
false positive: Arguments to CMD in exec for must not contain single quotes

### ISSUE THREAD

**Issue by russelltsherman** [RedCoolBeans/dockerlint#22]
### Issue details

dockerlint reports false positive when CMD includes whitespace between brackets and quotes.

### Steps to reproduce/test case

add spaces between brackets and quotes triggers this lint error
CMD [ "node" ]

### Affected version(s)

0.2.0

**Comment by cstrouse** [comment]
This also occurs without spaces between the bracket and the quotes.

```bash
CMD ["awesome_bot", "README.md", "--allow-dupe", "--allow-redirect", "--allow-ssl", "--allow-timeout", "--allow 999,403,405,500"]
```

**Comment by odivlad** [comment]
I also have false-positive because of comma symbol in argument, that I should pass to exec:

```
CMD ["my-executable", "-b", "a,1"]
```

I see, that it occurs because of split by comma, but I don't know how to fix it correctly...
https://github.com/RedCoolBeans/dockerlint/blame/master/src/checks.coffee#L84
